packages/story-editor/src/components/videoTrim/useVideoTrimMode.js

```diff
@@ -115,20 +115,17 @@ function useVideoTrimMode() {
     if (selectedElement?.type !== 'video' || !selectedElement?.resource) {
       return false;
     }
-    const { id, src, isExternal } = selectedElement.resource;
+
     return (
       isVideoTrimEnabled &&
       isTranscodingEnabled &&
-      src &&
-      !isBlobURL(src) &&
-      !isExternal &&
-      !isCurrentResourceUploading(id)
+      canTranscodeResource(selectedElement.resource)
```

Comment:
@miina @swissspidy I believe this line why the trim button is hidden now instead of disable on trimming. Trim mode need to enabled for the button to show to be disabled. This change needs to be reverted.